Update 01. Chats.md

SangeetaMishr opened the pull request:
Added description on chats and images

mahajantejas reviewed (commented):
Hey Sangeeta, 

can you please add the closing tag for the images in this documentation? 

all images should have an ending tag  />

SangeetaMishr commented:
@mahajantejas I have added the images and checked in preview it is appearing fine and also added headers and alignment of  text and images, request you to have a final look on it.

mahajantejas reviewed (changes requested):
can you pls add the image url tags
